packages/lit-dev-content/site/docs/tools/publishing.md

```diff
@@ -106,16 +106,26 @@ You can run Babel via a bundler plugin such as [@rollup/plugin-babel](https://ww
 
 The following are other good practices to follow when publishing reusable Web Components.
 
-### Do not import polyfills into modules
+### Don't import polyfills into modules
 
 Polyfills are an application concern, so the application should depend directly
 on them, not individual packages. The exact polyfills needed often depends on
 the browsers the application needs to support, and that choice is best left to
-the application developer using your component.
+the application developer using your component. Your component's documentation **should** clearly identify any APIs it uses that may require polyfills. 
 
 Packages may need to depend on polyfills for tests and demos, so if
 they're needed, they should only go in `devDependencies`.
 
+### Don't bundle, minify, or optimize modules
+
+Bundling and other optimizations are application concerns. Bundling a reusable component before publishing to npm can also introduce multiple versions of Lit (and other packages) into your users' application since npm can't deduplicate the packages. This causes bloat and may cause bugs.
```

Comment:
```suggestion
Bundling and other optimizations are application concerns. Bundling a reusable component before publishing to npm can also introduce multiple versions of Lit (and other packages) into a user's application since npm can't deduplicate the packages. This causes bloat and may cause bugs.
```